presto-main/src/test/java/com/facebook/presto/execution/TestSqlTask.java

```diff
@@ -102,6 +105,12 @@ public TestSqlTask()
                 new TaskManagerConfig());
     }
 
+    @BeforeMethod
+    public void setUp()
+    {
+        sqlTask = createInitialTask();
```

Comment:
I'm not sure if it is correct. We run tests in parallel.

Reply:
Could you elaborate on this? I don't understand why that would be a problem.

Reply:
`SqlTask` is state full. Since the test methods are run in parallel the `SqlTask` may endup in an inconsistent state resulting in test failures

Reply:
Should i leave everything in the `@BeforeClass` then?

Reply:
Is it correct to reuse the same sqlTask across all the test cases?